src/crimson/osd/recovery_backend.h

```diff
@@ -272,6 +272,14 @@ class RecoveryBackend {
   virtual interruptible_future<> handle_backfill_op(
     Ref<MOSDFastDispatchOp> m,
     crimson::net::ConnectionXcoreRef conn);
+
+  /**
+   * replica_push_targets
+   *
+   * Holds obc on replica for in-progress pushes, see
+   * ReplicatedRecoveryBackend::handle_push
+   */
+  std::map<hobject_t, crimson::osd::ObjectContextRef> replica_push_targets;
```

Comment:
~Could we have previously lose any in-progress obcs? Or is it introduced to more easily access these obcs?~ Ah, we want to ensure these (to be used) in-progress obc are still referenced somewhere IIUC. Cool!